.vib/supabase/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+http:
+  http://supabase-auth:{{ .Vars.auth.service.ports.http }}/health:
+    status: 200
+  http://supabase-meta:{{ .Vars.meta.service.ports.http }}/health:
+    status: 200
+  http://supabase-realtime:{{ .Vars.realtime.service.ports.http }}/:
+    status: 200
+  http://supabase-rest:{{ .Vars.rest.service.ports.http }}/:
+    status: 200
+  http://supabase-storage:{{ .Vars.storage.service.ports.http }}/status:
+    status: 200
+  http://supabase-studio:{{ .Vars.studio.service.ports.http }}/:
+    status: 200
+
+file:
+  /bitnami/kong/declarative-conf/kong.yml:
```

Comment:
I understand this check also doubles as a verification for the mounted volume `rendered-declarative-conf`. 

To ease future maintenance, we should try having the configuration actively tested included in the pipelines `runtime_parameters` and, ideally, parametrized via `vars.yaml`. In this case, I would add `kong.kong.extraVolumeMounts/extraEnvVars` to the run params and `/bitnami/kong/declarative-conf/kong.yml` as a Goss' var.